pyotgw/protocol.py

```diff
@@ -62,9 +63,17 @@ def connection_lost(self, exc):
         self.transport.close()
         for q in [self._cmdq, self._updateq]:
             while not q.empty():
-                q.get()
-        if self._watchdog_task is not None:
-            self._watchdog_task.cancel()
+                try:
+                    q.get_nowait()
+                except Empty:
```

Comment:
Same as above, use QueueEmpty from asyncio.queues

Reply:
Resolved in ac3c3ab